### DISCUSSION TITLE
Plots with missing keys

### DISCUSSION THREAD

**Discussion by Ne-maxim** [Chia-Network/chia-blockchain#14405] [Support]
Hi! After reinstallation system( Windows) i have a problem: working plots after full synchronization losted their keys and now is writing plots with missing keys. Can u help me? What should i do?

**Comment by shaneo257** [comment]
when you started chia for the first time, did you put in the correct/previous mnemonics?

**Reply by Ne-maxim** [reply]
Thank you!!! Problem was with the key, but i solve it
